### PR TITLE
removed DiscordUtils#cleanContent

### LINKED ISSUE #1172: Remove DiscordUtils since it no longer serves any purpose
**Bug description**

The only purpose of https://github.com/SwitchbladeBot/switchblade/blob/dev/src/utils/DiscordUtils.js was to polyfill the cleanContent method from Discord.js version 12, however, we are now using Discord.js v12 so we can just use their function instead of DiscordUtils.

### PR DESCRIPTION
**Pull request by almeidx**
Closes #1172

### CODE REVIEW

**Review by Doges — DISMISSED** [review]
LGTM

File: src/commands/government/searchplate.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const { Command, CommandError, SwitchbladeEmbed, Constants } = require('../../')
-const { cleanContent } = require('../../utils/DiscordUtils')
+const { Util } = require('discord.js')
```
Comment (jaxxibae) [inline src/commands/government/searchplate.js:2]:
```suggestion
const { cleanContent } = require('discord.js').Util
```

File: src/commands/government/searchplate.js [hunk 2]
```diff
@@ -18,15 +18,15 @@ module.exports = class searchPlate extends Command {
...
+          .setDescription(`**${t('commands:searchplate.plate')}**: ${plate.placa}\n**${t('commands:searchplate.model')}**: ${plate.marca} ${plate.modelo}\n**${t('commands:searchplate.year')}**: ${plate.anoModelo}\n**${t('commands:searchplate.color')}**: ${plate.cor}\n**${t('commands:searchplate.city')}**: ${plate.municipio} - ${plate.uf}\n**${t('commands:searchplate.status')}**: ${(plate.situacao !== 'Roubo/Furto') ? t('commands:searchplate.notstolen') : `**${t('commands:searchplate.stolen')}**`}\n**${t('commands:searchplate.chassis')}**: ${Util.cleanContent(plate.chassi, message)}\n\n**${t('commands:searchplate.image')}**:`)
```
Comment (jaxxibae) [inline src/commands/government/searchplate.js:29]:
```suggestion
          .setDescription(`**${t('commands:searchplate.plate')}**: ${plate.placa}\n**${t('commands:searchplate.model')}**: ${plate.marca} ${plate.modelo}\n**${t('commands:searchplate.year')}**: ${plate.anoModelo}\n**${t('commands:searchplate.color')}**: ${plate.cor}\n**${t('commands:searchplate.city')}**: ${plate.municipio} - ${plate.uf}\n**${t('commands:searchplate.status')}**: ${(plate.situacao !== 'Roubo/Furto') ? t('commands:searchplate.notstolen') : `**${t('commands:searchplate.stolen')}**`}\n**${t('commands:searchplate.chassis')}**: ${cleanContent(plate.chassi, message)}\n\n**${t('commands:searchplate.image')}**:`)
```

File: src/structures/command/parameters/types/StringParameter.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const Parameter = require('./Parameter.js')
-const DiscordUtils = require('../../../../utils/DiscordUtils.js')
 const CommandError = require('../../CommandError.js')
+const { Util } = require('discord.js')
```
Comment (jaxxibae) [inline src/structures/command/parameters/types/StringParameter.js:3]:
```suggestion
const { cleanContent } = require('discord.js').Util
```

File: src/structures/command/parameters/types/StringParameter.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ module.exports = class StringParameter extends Parameter {
     arg = arg ? (typeof arg === 'string' ? arg : String(arg)) : undefined
     if (!arg) return
 
-    if (this.clean) arg = DiscordUtils.cleanContent(arg, message)
+    if (this.clean) arg = Util.cleanContent(arg, message)
```
Comment (jaxxibae) [inline src/structures/command/parameters/types/StringParameter.js:21]:
```suggestion
    if (this.clean) arg = cleanContent(arg, message)

```

**Review by jaxxibae — CHANGES_REQUESTED** [review]

**Comment by jaxxibae** [comment]
PS: Test before making changes, because I'm not 100% sure it'll work

**Review by Doges — APPROVED** [review]

**Review by davipatury — APPROVED** [review]
LGTM

**Pull request merged**